src/host/ut_host/TextBufferTests.cpp

```diff
@@ -2988,13 +2988,7 @@ void TextBufferTests::ReflowPromptRegions()
             til::point afterPos = originalPos;
             // walk that original pos dx times into the actual real place in the buffer.
             auto bufferViewport = tb.GetSize();
-            const auto walkDir = Viewport::WalkDir{ dx < 0 ? Viewport::XWalk::LeftToRight : Viewport::XWalk::RightToLeft,
-                                                    dx < 0 ? Viewport::YWalk::TopToBottom : Viewport::YWalk::BottomToTop };
-            for (auto i = 0; i < std::abs(dx); i++)
-            {
-                bufferViewport.WalkInBounds(afterPos,
-                                            walkDir);
-            }
+            bufferViewport.WalkInBounds(afterPos, -dx);

```

Comment:
@zadjii-msft Here's why the tests failed... I previously had this as:
```cpp
bufferViewport.WalkInBounds(afterPos, dx);
```
but on closer inspection of the old code I noticed that the `dx` is meant inverse to an offset. I don't quite understand the old code to say why that is, but it does work now by simply using `-dx`.

I guess I'm being overly defensive over my PR here, but I do think it's fairly solid after all. 😅 (To be clear I don't mind merging it in 1.22 only.)